.github/scripts/queries-validator/metadata-schema.json

```diff
@@ -42,6 +42,17 @@
                 "INFO"
             ]
         },
+        "oldSeverity": {
+            "type": "string",
+            "enum": [
+                "TRACE",
+                "CRITICAL",
```

Comment:
```suggestion
```

Reply:
@pereiramarco011 _CRITICAL_ will never be an "old severity".

Reply:
done (here)[https://github.com/Checkmarx/kics/pull/6959/commits/779abaa3155e47485af6e50b11b198a966d86a18]